packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -135,6 +135,7 @@ import {
   commitAttachRef,
   commitDetachRef,
 } from './ReactFiberCommitWork';
+import * as dispatcher from './ReactFiberDispatcher';
```

Comment:
We should avoid exporting everything. Make it explicit what you want this object to contain. Too easy to accidentally bring in too much. For example, I think the resulting object that rollup produces for this is not what you expect.

Reply:
I figured it was ok in this case because I need to use an object for dynamic dispatch. But I agree explicit is better.